cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/adls/ADLSStorageClient.java

```diff
@@ -0,0 +1,110 @@
+package com.linkedin.openhouse.cluster.storage.abs;
+
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.linkedin.openhouse.cluster.storage.StorageClient;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import com.linkedin.openhouse.cluster.storage.configs.StorageProperties;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.iceberg.azure.adlsv2.ADLSFileIO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * ABSStorageClient is an implementation of the StorageClient interface for Azure Blob Storage. It
+ * uses the {@link BlobClient} to interact with Blob Storage.
+ */
+@Slf4j
+@Lazy
+@Component
+public class ADLSStorageClient implements StorageClient<DataLakeFileClient> {
+
+  private static final StorageType.Type ADLS_TYPE = StorageType.ADLS;
+
+  @Autowired private StorageProperties storageProperties;
+
+  private DataLakeFileClient dataLake;
+
+  private static final String DEFAULT_ENDPOINT = "abfs:/";
```

Comment:
endpoint needs to be= "abfs://"
default_rootpath/bucket can be = "tmp"

reason: we want all object stores to have this pattern so that provisioning bucket becomes easier

Reply:
Why do you need to configure these defaults here?